README.md

```diff
@@ -9,10 +9,16 @@ Here we define the prometheus-agent chart with its templates and default configu
 
 **What is this app?**
 
+This app installs a Prometheus CR bundled with all required resources(RBAC, SA, NetworkPolicy...) in WC.
```

Comment:
I'd rather say something like:

What:
> Prometheus-agent is a lightweight prometheus instance, that forwards metrics to a prometheus via remotewrites and does not manage storage of data.
> See https://prometheus.io/blog/2021/11/16/agent/ for more context.

Why:
> We use it to gather metrics on WCs, and remoteWrite them to prometheus on the MC.
> This way we support features of Prometheus (like serviceMonitors) on the WCs, without actually hosting it there.

Who:
> GiantSwarm deploys prometheus-agent to all workload clusters, in the `giantswarm` namespace.


Then we should add some more context around:
- what are the cases when customers would want to install their own prometheus-agent, and when they could use ours?
- how to make sure different prometheus-agents don't conflict? That their prometheus resources (like servicemonitors) would be used by their agent, not ours?
- prometheus-agent CRDs, that customers should not re-install
- are there limitations with their own prometheus-agent deployment? Like, "you should not enable node-exporters (they are disabled in default values, it would conflict with ours)"

Reply:
We can have a small pairing session on those!

Reply:
@hervenicol you can check